Week-2/Html Advanced/lineUp.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Select Line Up</title>
+</head>
+<body>
+    <form>
+        <p>
+            <fieldset>
+                <legend>Status:</legend>
+                <label >
+                    <input type="radio" name="status" value="Draft">
+                    Draft
+                </label>
+                <label>
+                    <input type="radio" name="status" value="Final" checked>
+                    Final
+                </label>
+            </fieldset>
+        </p>
+
+        <p>
+            <label for="gameDate">Date: </label>
+            <select name="gameDate" id="gameDate">
+                <option value="March 20">March 20</option>
```

Comment:
Use proper date formats: The date values in the dropdown list are in a format like "March 20." While this is understandable for most users, you can make it even more accessible by including the full year (e.g., "March 20, 2023") to prevent ambiguity, especially if the date spans multiple years.